src/nxt_time_parse.c

```diff
@@ -448,15 +442,6 @@ nxt_term_parse(const u_char *p, size_t len, nxt_bool_t seconds)
             scale = 1;
             break;
 
-        case ' ':
```

Comment:
It's used by input like `1  ` which ends with spaces.

Reply:
Hmm, OK, in that case I'll add a test case for it...

Reply:
That would be great.